app/app/Http/Controllers/SignUpController.php

```diff
@@ -71,14 +77,25 @@ public function createUser(Request $request)
 
     public function confirmEmail($user_email, $email_verification_token)
     {
+        $after_signup_redirect = Input::get('after_signup_redirect');
         $email = $user_email;
         $confirmCode = $email_verification_token;
         if (BaseUser::confirmEmail($email, $confirmCode)) {
-            return view('pages.signup.success', [
-                'email' => $email,
+             if (Input::has('after_signup_redirect')) {
+                return view('pages.signup.success', [
+                'confirmmessage' => 'Your email has been confirmed.',
+                'can_sign_in' => true,
+                'redirect' => '/signin?after_signin_redirect=/add-location'
+                ]);
+ 
+            }
+            else{
```

Comment:
Did you run 'composer all'?  I have a feeling that else statement would fail the coding style tests.